pkgs/applications/misc/clipqr/default.nix

```
@@ -0,0 +1,54 @@
+{ buildGoModule, copyDesktopItems, fetchFromGitLab, lib, makeDesktopItem, pkgs, pkgconfig, ... }:
+
+buildGoModule rec {
+  pname = "clipqr";
+  version = "1.0.0";
+
+  src = fetchFromGitLab {
+    owner = "imatt-foss";
+    repo = "clipqr";
+    rev = "v${version}";
+    sha256 = "sha256-E90nTJtx4GOacu8M7oQBznnSQVDIZatibgKMZEpTUaQ=";
+  };
+
+  vendorSha256 = "5kAOSyVbvot4TX/XfRNe1/zZk6fa7pS1Dvn9nz11u3U=";
+
```

Please add strip flags:
```suggestion

ldflags = [ "-s" "-w" ]

```